src/main/java/co/rsk/federate/btcreleaseclient/cache/PegoutSignedCacheImpl.java

```diff
@@ -41,14 +51,19 @@ public void putIfAbsent(Keccak256 pegoutCreationRskTxHash) {
         .ifPresent(rskTxHash -> cache.putIfAbsent(rskTxHash, Instant.now()));
   }
 
+  void performCleanup() {
+    cache.entrySet().removeIf(
+        entry -> !hasTimestampNotExpired(entry.getValue()));
```

Comment:
This looks like a negative condition. Should we do something like:

```java
cache.entrySet().removeIf(entry -> hasExpired(entry.getValue()));
```
Or:

```java
cache.entrySet().removeIf(entry -> hasExpired(entry));
```

Reply:
We already use this method in the same class inside the `hasAlreadyBeenSigned` method. The idea is that we reuse the same logic to be consistent, even if it thats means negating the conditional. Wdyt?

Reply:
I agree with jere. 
We can create another method, `hasExpiredTimestamp` that simply returns `!hasTimestampNotExpired`.
But we shouldn't have negative conditions.

Reply:
I vote for just a variable. `hasExpire` looks good to me.

Reply:
I am a bit confused, what would it look like in a code snippet? :)

Reply:
Updated to what we discussed offline :)